### ISSUE TITLE
[Bug]: assertNoAccessibilityIssues does not report any violations

### ISSUE THREAD

**Issue by lucadegasperi** [pestphp/pest#1491]
### What Happened

I have created a fresh Laravel Project and installed Pest v4 with the browser testing plugin. I tried calling the  `assertHasNoAccessibilityIssues()` function but it returns no a11y violations, which is weird. When running the `window.axe.run()` command in the browser and awaiting the execution I do get a couple of violations

Am I missing something?

### How to Reproduce

```php
<?php

it('may welcome the user', function () {
    $page = visit('/');

    $page->assertNoAccessibilityIssues();

});
```

run `php artisan test` and get no results

modify the test to 

```php
<?php

it('may welcome the user', function () {
    $page = visit('/');

    $page->assertNoAccessibilityIssues();

    $page->debug();

});
```

and once the browser opens, open the console and run: `window.axe.run().then(result => console.log(result.violations))` and see the violations which pest does not report.

### Pest Version

4.0

### PHP Version

8.4

### Operation System

macOS
